public/app/features/dashboard-scene/serialization/transformSaveModelToScene.test.ts

```diff
@@ -144,7 +144,6 @@ describe('transformSaveModelToScene', () => {
 
       const scene = createDashboardSceneFromDashboardModel(oldModel);
 
-      expect(scene.state.$behaviors).toHaveLength(6);
```

Comment:
Removing this assertion since I think the length of the `$behaviors` array is sort of independent of whether or not the cursor sync behavior is actually applied, which is what the next assertion does anyway. Plus it means we don't have to update this test every time we make add/remove a behavior to the dashboard scene